[ci] Manual npm publish workflow failed on Husky precommit

Running the job to manually release the packages resulted in an error where the job failed because of the `prepare-commit-msg`. See following thread: https://github.com/lerna/lerna/issues/2664

Possible solution: 

```bash
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

if [ "$2" != "message" ];then 
    exec < /dev/tty && node_modules/.bin/cz --hook "$1" "$2" "$3" || true
fi
```

Related job: https://github.com/layer5io/sistent/actions/runs/6426918291/job/17451838818
```bash
/home/runner/work/_temp/5de2d954-1dbb-475f-8357-8ec32f9c2c1d.sh: line 4: [: =: unary operator expected
lerna notice cli v7.3.0
lerna info versioning independent
lerna info ci enabled
lerna info Assuming all packages changed
lerna info getChangelogConfig Successfully resolved preset "conventional-changelog-angular"

Changes:
lerna info auto-confirmed 
 - @layer5/sistent-components: 0.3.0 => 0.4.0
lerna info execute Skipping releases
 - @layer5/rtk-query-codegen: 0.0.1 => 0.1.0
 - @layer5/sistent-svg: 0.6.6 => 0.7.0

lerna ERR! Error: Command failed with exit code 1: git commit -F /tmp/e10f445f-2816-4[24](https://github.com/layer5io/sistent/actions/runs/6426918291/job/17451838818#step:5:25)d-b632-8f8801d6d0bf/lerna-commit.txt
lerna ERR! Checking formatting...
lerna ERR! All matched files use Prettier code style!
lerna ERR! [STARTED] Preparing lint-staged...
lerna ERR! [COMPLETED] Preparing lint-staged...
lerna ERR! [STARTED] Running tasks for staged files...
lerna ERR! [STARTED] package.json — 7 files
lerna ERR! [STARTED] *.{ts,tsx,md} — 3 files
lerna ERR! [STARTED] eslint --fix
lerna ERR! [COMPLETED] eslint --fix
lerna ERR! [STARTED] yarn format:write
lerna ERR! [COMPLETED] yarn format:write
lerna ERR! [COMPLETED] *.{ts,tsx,md} — 3 files
lerna ERR! [COMPLETED] package.json — 7 files
lerna ERR! [COMPLETED] Running tasks for staged files...
lerna ERR! [STARTED] Applying modifications from tasks...
lerna ERR! [COMPLETED] Applying modifications from tasks...
lerna ERR! [STARTED] Cleaning up temporary files...
lerna ERR! [COMPLETED] Cleaning up temporary files...
lerna ERR! .husky/prepare-commit-msg: 4: cannot open /dev/tty: No such device or address
lerna ERR! husky - prepare-commit-msg hook exited with code 2 (error)
lerna ERR!     at makeError (/home/runner/work/sistent/sistent/node_modules/execa/lib/error.js:60:11)
lerna ERR!     at handlePromise (/home/runner/work/sistent/sistent/node_modules/execa/index.js:118:[26](https://github.com/layer5io/sistent/actions/runs/6426918291/job/17451838818#step:5:27))
lerna ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
lerna ERR!     at async VersionCommand.gitCommitAndTagVersionForUpdates (/home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:8346:11)
lerna ERR!     at async VersionCommand.commitAndTagUpdates (/home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:83[28](https://github.com/layer5io/sistent/actions/runs/6426918291/job/17451838818#step:5:29):18)
lerna ERR!     at async Promise.all (index 0)
lerna ERR! lerna Command failed with exit code 1: git commit -F /tmp/e10f445f-2816-424d-b6[32](https://github.com/layer5io/sistent/actions/runs/6426918291/job/17451838818#step:5:33)-8f8801d6d0bf/lerna-commit.txt
lerna ERR! lerna Checking formatting...
lerna ERR! lerna All matched files use Prettier code style!
lerna ERR! lerna [STARTED] Preparing lint-staged...
lerna ERR! lerna [COMPLETED] Preparing lint-staged...
lerna ERR! lerna [STARTED] Running tasks for staged files...
lerna ERR! lerna [STARTED] package.json — 7 files
lerna ERR! lerna [STARTED] *.{ts,tsx,md} — 3 files
lerna ERR! lerna [STARTED] eslint --fix
lerna ERR! lerna [COMPLETED] eslint --fix
lerna ERR! lerna [STARTED] yarn format:write
lerna ERR! lerna [COMPLETED] yarn format:write
lerna ERR! lerna [COMPLETED] *.{ts,tsx,md} — 3 files
lerna ERR! lerna [COMPLETED] package.json — 7 files
lerna ERR! lerna [COMPLETED] Running tasks for staged files...
lerna ERR! lerna [STARTED] Applying modifications from tasks...
lerna ERR! lerna [COMPLETED] Applying modifications from tasks...
lerna ERR! lerna [STARTED] Cleaning up temporary files...
lerna ERR! lerna [COMPLETED] Cleaning up temporary files...
lerna ERR! lerna .husky/prepare-commit-msg: 4: cannot open /dev/tty: No such device or address
lerna ERR! lerna husky - prepare-commit-msg hook exited with code 2 (error)
Error: Process completed with exit code 1.
```

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
  - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/.husky/prepare-commit-msg
+++ b/.husky/prepare-commit-msg
@@ -1,4 +1,6 @@
 #!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
+. "$(dirname "$0")/_/husky.sh"
 
-exec < /dev/tty && cz --hook || true
+if [ "$2" != "message" ];then 
+    exec < /dev/tty && cz --hook "$1" "$2" "$3" || true
+fi

